src/bot.py

```diff
@@ -528,6 +531,15 @@ def get_top_10(data):
     """
     await custom_meme.ify(ctx, face_detection.tim_scale, 'memes/tim/tim-face.png', 'timify.png', args)
 
+@client.command(name='lanify', aliases = ['lan-ify'])
+async def lanify(ctx, *args):
+    """Command to paste lan's face on top of faces in an inputed image using facial recognition
+
+       Args:
+        - ctx: context that the command occured use this to access the message and other attributes
+    """
+    await ify(ctx, tim_scale, 'memes/lan/lan-face.png', 'lan-face.png', args)
```

Comment:
Does not work, functions and variables from other files need to be referenced. Should be:
```suggestion
    await custom_meme.ify(ctx, face_detection.lan_scale, 'memes/lan/lan-face.png', 'lan-face.png', args)
```
Also please add this scale underneath `tim_scale` in _face_detection.py_
`lan_scale = (1.55, 0.45, 0.4)`